packages/mattermost-redux/src/selectors/entities/preferences.ts

```diff
@@ -216,6 +216,10 @@ export function getAutoTourTreatment(state: GlobalState): AutoTourTreatments | u
     return getFeatureFlagValue(state, 'AutoTour') as AutoTourTreatments | undefined;
 }
 
+export function getCreateGuidedFirstChannelCTATreatment(state: GlobalState): boolean {
+    return getFeatureFlagValue(state, 'GuidedChannelCreation') === 'true';
+}
```

Comment:
Given that we don't actually return the treatment but if we should add do the channel creation then I would drop `Treatment` fromt he name
```suggestion
export function getCreateGuidedChannel(state: GlobalState): boolean {
    return getFeatureFlagValue(state, 'GuidedChannelCreation') === 'true';
}
```

Reply:
ok!